### ISSUE TITLE
[BUG]Newest Version Does Not Work!

### ISSUE THREAD

**Issue by DeMoN100** [guardianproject/orbot#1227]
Pixel5a
Calyxos 5.14.0 Android 14.

The newest version does not work says configuration invalid on fresh install & update. I wanted to revert to the previous version but it was removed from this repo so now I am forced to use an older version 16.6.4 rc 1 tor 0.4.7.11 for it to work. The downside to this is I can no longer have my vpn & orbot on at the same time since this older version only orbots in vpn mode.

Please add back all versions between this one & the latest one of them seemed to work for me & allowed me to use orbot with my vpn on at the same time. Also this newer UI on the later versions the start orbot button has never worked on my device the only way I was able to start orbot was through the notification when it did work.

**Comment by n8fr8** [comment]
@DeMoN100 you can enable power user mode in Orbot settings. that will allow you to use it without the VPN feature.

**Comment by DeMoN100** [comment]
Yes I can do this & the button now works but the connect button should have worked without me having to do that. The connect button in notifications & the app did nothing when that setting was off.

**Comment by bitmold** [comment]
This version is very obsolete, closing issue.
